_docs/configuration_files/pipeline_json/image.rest

```diff
@@ -3,6 +3,15 @@
 
 Holds settings for the baked image
 
+``bake_instance_type``
+**********************
+
+Defines the instance type for Rosco (bake step) to use. This could help with issues
+of large and complex bakes. 
```

Comment:
Do we want to add the link to ec2info like we do for `instance_type`?

Reply:
Linking to official AWS docs.